torch/_refs/__init__.py

```diff
@@ -2560,6 +2560,18 @@ def broadcast_tensors(*tensors) -> List[TensorLikeType]:
     return list(_maybe_broadcast(*tensors, preserve_cpu_scalar_tensors=False))
 
 
+@aten.unsafe_chunk.default.py_impl(DispatchKey.CompositeImplicitAutograd)
```

Comment:
why does this go to _refs?

Reply:
This was suggested by @ezyang, I am not too sure about the reasoning :)

Reply:
oh oops, I either told you the wrong thing, or it got garbled in transit. Put it in _decompositions.

Reply:
ok updated :)